### PR TITLE
Jet wrappers

### PR DESCRIPTION
**Pull request by uncomputable**
Update libsimplicity and use generated jet wrappers.

### CODE REVIEW

File: simplicity-sys/vendor-simplicity.sh [hunk 1]
```diff
@@ -1,45 +1,47 @@
...
+echo "$HEAD" >> "$ORIGINAL_PATH/$DEPEND_PATH/simplicity-HEAD-revision.txt"
 
-# Only get the C directory. move C into parent folder, and then move it back
-mv "${DIR}/C" "./${DIR}_C"
-rm -rf "${DIR}"
-mv "${DIR}_C" "${DIR}"
+# Copy C folder to simplicity-sys/depend/simplicity
+# Use rsync to copy only files tracked by git
+cd C
+git ls-files | rsync -av --files-from=- . "$ORIGINAL_PATH/$DEPEND_PATH/$VENDORED_SIM_DIR"
```
Comment (apoelstra) [inline simplicity-sys/vendor-simplicity.sh:40]: In 089847766a3ce08527511548c27a67895cb8d691:

What do you mean "only files tracked by git"? rsync does not understand git and you haven't given it any interesting filtering options.

Comment (apoelstra) [reply simplicity-sys/vendor-simplicity.sh:40]: Oh, derp, you're piping `ls-files` into `rsync`. Interesting.

I think I'm OK with the rsync dependency then.

File: simplicity-sys/vendor-simplicity.sh [hunk 2]
```diff
@@ -1,45 +1,47 @@
 #!/bin/bash
```
Comment (apoelstra) [inline simplicity-sys/vendor-simplicity.sh:1]: In 089847766a3ce08527511548c27a67895cb8d691:

If you want this to be nix-compatible we have to replace `/bin/bash` with `/usr/bin/env bash`.

**Review by apoelstra — APPROVED** [review]
ACK 6a7684197f119c1fc15d9eff7454e394adb599ce

**Comment by uncomputable** [comment]
I fixed the bash shebang. I also fixed the header of the autogenerated HEAD revision, which was different depending on the location of the vendor script relative to pwd.

**Review by apoelstra — APPROVED** [review]
ACK c9d5c00bb8d6bdbaf7d328ad00a11af3f6e12a3c

**Pull request merged**